### PR TITLE
feat(exposition): add `auth:claim` directive

### PR DESCRIPTION
**Pull request by temich**

### CODE REVIEW

File: extensions/exposition/components/identity.federation/manifest.toa.yaml [hunk 1]
```diff
@@ -44,6 +44,10 @@ operations:
     output:
       identity:
         id: string
+        claim:
+          iss: string
+          aud: string
+          sub: string
```
Comment (tinovyatkin) [inline extensions/exposition/components/identity.federation/manifest.toa.yaml:50]: Don't we want to start using JSON schema for declarations?

File: extensions/exposition/components/identity.federation/manifest.toa.yaml [hunk 2]
```diff
@@ -44,6 +44,10 @@ operations:
     output:
       identity:
         id: string
+        claim:
+          iss: string
+          aud: string
```
Comment (tinovyatkin) [inline extensions/exposition/components/identity.federation/manifest.toa.yaml:49]: We have `aud` as array of strings in the trust configuration, what would be this single string - a matched audience?

Comment (temich) [reply extensions/exposition/components/identity.federation/manifest.toa.yaml:49]: https://github.com/toa-io/toa/pull/656/commits/b538908c4e1a0fbe6dae5140adfcd561c8f0382b

File: extensions/exposition/components/identity.federation/manifest.toa.yaml (hunk 1 above)
Comment (temich) [reply extensions/exposition/components/identity.federation/manifest.toa.yaml:50]: Yes, but next time.

**Pull request merged**